EnglishTranslation-master/ITEM.tsv

```diff
@@ -10199,37 +10199,37 @@ ITEM_20160224_010198	Material of a rune stone. In order to become a Rune Caster,
-ITEM_20160310_010222	With every new piece of equipment, the power bestowed by the goddess of earth is newly imparted on it.
-ITEM_20160310_010223	Users of this equipment may not realize, but designers and artisans know obtaining the blessing of Goddess Zemyna for using leather is not as easy as for cloth or metal. In any case, thanks to their efforts, today it shines on the battelfield.
-ITEM_20160310_010224	Because it comes into direct contact with the ground, it is more readily blessed by Goddess Zemyna, compared to other leather equipment.
-ITEM_20160310_010225	Some Lolopanther equipment was also made using the power not of Goddess Zemyna, master of the tower, but of the invading demons. In such cases, equipment balance is important.
-ITEM_20160310_010226	Who was behind such efforts to take the demons' power and what they targeted remains unknown, buried under the history of the Earth Tower.
-ITEM_20160310_010227	The goddess of earth is glad offer humans the ores forged at Her hands. However, She does not look kindly on those who carelessly use them and easily discard of them.
-ITEM_20160310_010228	The goddess of earth may have allowed humans to carry weapons; that does not mean She hopes they will bring war and bloodshed. There is a special reason to make use of this particular piece of equipment.
+ITEM_20160310_010215	Warsis Creak has succeeded in developing a new type of weapon from the Lolopanther Musket. However, it is unknown when and how this happened.
+ITEM_20160310_010216	A weapon firm as the ground, light as a feather and strong as a rock.
+ITEM_20160310_010217	The masters who deal with gunpowder regularly use it with newly designed or discovered weapons. The same can be said for this cannon. Quicktongue, however, hardly shares any new discoveries and is likely considering his military career.
+ITEM_20160310_010218	Those who dare to belittle the weapons grown by the goddess are doomed to have their blood fed to her earth.
+ITEM_20160310_010219	Those who worship the great goddess will receive wisdom of humility. This equipment proves that this is true not only for clerics, but for wizards too.
+ITEM_20160310_010220	Whoever carries this will become the master of the earth and receive its grace with every step. However, maintaining the cost of this equipment is of course a problem for humans and unrelated to the goddess.
+ITEM_20160310_010221	Regardless of what rights the goddess of the earth may give, those who carry and use this equipment do so with their own abilities. Do you not wonder whether it possesses an even greater power?
+ITEM_20160310_010222	With every new piece of equipment, the power bestowed by the goddess of the earth is newly imparted on it.
+ITEM_20160310_010223	Users of this equipment may not realize, but designers and artisans know that obtaining the blessing of Goddess Zemyna for using leather is not as easy as for cloth or metal. But, thanks to their efforts, it shines today on the battlefield.
```

Comment:
`obtaining the blessing of Goddess Zemyna for using leather is not as easy as it is for using cloth or metal. But thanks to their efforts`